embed_grader/serapis/forms/course_forms.py

```diff
@@ -119,14 +119,19 @@ class CourseEnrollmentForm(Form):
     error_messages = {
         'course_already_enrolled': "You have already enrolled in this course.",
     }
-    
+
     def __init__(self, *args, **kwargs):
         self.user = kwargs.pop('user')
         super(CourseEnrollmentForm, self).__init__(*args, **kwargs)
 
-        current_year = timezone.now().year
+        now = timezone.now()
+        cur_year = now.year
+        cur_month = now.month
+        cur_quarter = (cur_month - 1) // 3
+        cur_year_quarter = cur_year * 4 + cur_quarter
```

Comment:
This line seems redundant

Reply:
which line is redundant exactly?

Reply:
131

Reply:
oh right, it can be removed.

Reply:
👍

Reply:
I have just added a description of the test.

Reply:
The change was tested by creating a course in Winter 2018 and a course in Summer 2018, and then attempting to enroll. In the dropdown, only the Winter 2018 course appeared.